### ISSUE TITLE
Unable to run optimized BERT: model shape expected by framework [-1,-1] doesn't match model configuration shape [-1,-1]

### ISSUE THREAD

**Issue by lapolonio** [NVIDIA/TensorRT#171]
After following the demo/BERT [instructions](https://github.com/NVIDIA/TensorRT/tree/master/demo/BERT#example-workflow). When deploying the optimized model on TRTIS I get:

> INVALID_ARGUMENT: getPluginCreator could not find plugin CustomEmbLayerNormPluginDynamic version 1
safeDeserializationUtils.cpp (259) - Serialization Error in load: 0 (Cannot deserialize plugin since corresponding IPluginCreator not found in Plugin Registry)

**TRTIS Version:** nvcr.io/nvidia/tensorrtserver:19.09-py3
**Built engine on:** nvcr.io/nvidia/tensorflow:19.09-py3

I run the server: `LD_PRELOAD=/opt/tensorrtserver/libbert_plugins.so:/opt/tensorrtserver/libcommon.so trtserver --model-repository=/models --strict-model-config=false`

I just copied the files from TensorRT/demo/BERT/build

**Comment by rmccorm4** [comment]
Seems this is cross-posted in TRTIS and solved there: https://github.com/NVIDIA/tensorrt-inference-server/issues/786

TLDR: https://github.com/NVIDIA/tensorrt-inference-server/issues/786#issuecomment-546547881

**Comment by rmccorm4** [comment]
Closing as with TRTIS 19.10 / v1.7 release, `--strict-model-config=false` for now works correctly for TensorRT models with variable-sized inputs and/or outputs.

Release notes: https://github.com/NVIDIA/tensorrt-inference-server/releases/tag/v1.7.0
